images/modsecurity/nginx.conf

```diff
@@ -0,0 +1,16 @@
+events {
+    worker_connections 1024;
+}
+
+http {
+    default_type 'text/plain';
+    server {
+        listen 8080;
+        server_name _;
+
+        location / {
```

Comment:
@aledbf my plan is as follows:

Testing:
1. Create a Docker Image containing nginx which just serves a modsecurity file. 
2. Move the image to the cluster
3. Expose the application as a service
4. Add the ModSecurity Remote Rules annotation pointing at our server.
5. Make sure it doesn't crash

Code:
1. Create a template function which sends a request to the server
2. If it fails then don't load the template

Let me know if you have any implementation details or if you are ok with the above.